src/main/java/edu/harvard/iq/dataverse/DatasetPage.java

```diff
@@ -371,7 +371,7 @@ public String getThumbnailString() {
         }
 
         if (!readOnly) {
-            DatasetThumbnail datasetThumbnail = dataset.getDatasetThumbnail();
+            DatasetThumbnail datasetThumbnail = dataset.getDatasetThumbnail(ImageThumbConverter.DEFAULT_DATASETLOGO_SIZE);
```

Comment:
Attn @landreev ... there is another issue that represents additional thumbnail improves that might cover this ... Dataset Thumbnail API - Size Parameter #5679.

Reply:
I'm not sure I understand the question, but I think the answer is that code is doing the right thing as is.

Reply:
I mean, it's not even clear which size to consider the "default", for the datasets in particular, anymore. Since all these different parts of the application now need different size thumbnail, it actually felt confusing to have a size-less method for just one of them. And I don't think it would keep things simpler/cleaner overall.

Reply:
Thank you for the discussion, @landreev and @scolapasta. Any pertinent details of the decisions here should be moved to #5679. Resolving.